Makefile

```diff
@@ -130,6 +130,11 @@ help:
 	} \
 	{ lastLine = $$0 }' $(MAKEFILE_LIST)
 
+## Generate README.md with Terradoc
+.PHONY: terradoc
```

Comment:
this is a duplicate, the target exists in the Makefile already - please remove

Reply:
good catch... please remove again